source/materials/OpticalMaterialProperties.cc

```diff
@@ -528,17 +528,18 @@ namespace opticalprops {
       //G4cout << "* GAr energy: " << std::setw(6) << sc_energy[i]/eV << " eV  ->  "
       //       << std::setw(6) << intensity[i] << G4endl;
     }
-    mpt->AddProperty("FASTCOMPONENT", sc_energy, intensity, sc_entries);
-    mpt->AddProperty("SLOWCOMPONENT", sc_energy, intensity, sc_entries);
-    mpt->AddProperty("ELSPECTRUM",    sc_energy, intensity, sc_entries);
+    mpt->AddProperty("SCINTILLATIONCOMPONENT1", sc_energy, intensity, sc_entries);
+    mpt->AddProperty("SCINTILLATIONCOMPONENT2", sc_energy, intensity, sc_entries);
+    mpt->AddProperty("ELSPECTRUM"             , sc_energy, intensity, sc_entries, 1);
 
     // CONST PROPERTIES
     mpt->AddConstProperty("SCINTILLATIONYIELD", sc_yield);
-    mpt->AddConstProperty("FASTTIMECONSTANT",   6.*ns);
-    mpt->AddConstProperty("SLOWTIMECONSTANT",   37.*ns);
-    mpt->AddConstProperty("YIELDRATIO",         .52);
+    mpt->AddConstProperty("SCINTILLATIONTIMECONSTANT1",   6.*ns);
+    mpt->AddConstProperty("SCINTILLATIONTIMECONSTANT2",   37.*ns);
+    mpt->AddConstProperty("SCINTILLATIONYIELD1", .342);
+    mpt->AddConstProperty("SCINTILLATIONYIELD2", .657);
```

Comment:
The sum of `SCINTILLATIONYIELD1` and `SCINTILLATIONYIELD2` should give 1, right? Therefore, I would change the second to `.658` (which is the rounding I find from the calculation).